provider/vsphere/vm_template_test.go

```diff
@@ -0,0 +1,234 @@
+// Copyright 2021 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package vsphere_test
+
+import (
+	"context"
+	"time"
+
+	"github.com/juju/clock/testclock"
+	"github.com/juju/juju/environs"
+	"github.com/juju/juju/environs/imagemetadata"
+	"github.com/juju/juju/provider/vsphere"
+	"github.com/juju/juju/provider/vsphere/internal/ovatest"
+	"github.com/juju/juju/provider/vsphere/internal/vsphereclient"
+	coretesting "github.com/juju/juju/testing"
+	"github.com/juju/testing"
+	jc "github.com/juju/testing/checkers"
+	"github.com/vmware/govmomi/object"
+	"github.com/vmware/govmomi/vim25/mo"
+	"github.com/vmware/govmomi/vim25/types"
+	gc "gopkg.in/check.v1"
```

Comment:
"github.com/juju/juju/..." should be in the last stanza, everything else in the selection, the 2nd stanza